GliaWidgets/Sources/Download/FileDownload.swift

```diff
@@ -28,7 +28,7 @@ class FileDownload {
 
         var accessibilityString: String {
             switch self {
-            case .none: return "none"
+            case .none: return "download"
```

Comment:
Is there a reason for the deviation between the case and the accessbility string?